x-pack/plugins/enterprise_search/server/lib/enterprise_search_config_api.test.ts

```diff
@@ -21,7 +21,7 @@ describe('callEnterpriseSearchConfigAPI', () => {
     accessCheckTimeoutWarning: 100,
   };
   const mockRequest = {
-    url: { path: '/app/kibana' },
+    url: { pathname: '/app/kibana' },
```

Comment:
This looks fine from the Enterprise Search's team end - although after a bit of digging we realized we don't actually need this `url` property at all in our code/tests, we just need the `headers` prop 🤦‍♀️ If it's easier for you all for any URL future changes, feel free to remove this line entirely in this PR.

Adding a suggestion for easy committing, but totally up to you as to whether you want to use it or not.

```suggestion
```

Reply:
Thanks for checking. It doesn't make much difference to me, but I'm a big fan of simplifying things, so I've added your suggestion 👍